examples/webgl_morphtargets_sphere.html

```diff
@@ -54,11 +56,11 @@
 
 				clock = new THREE.Clock();
 
-				const light1 = new THREE.PointLight( 0xff2200, 0.7 );
+				const light1 = new THREE.PointLight( 0xff2200, 50000 );
```

Comment:
I guess these high candela values are required since the point lights are far away from the sphere. Wouldn't it be better to move the point lights closer to the sphere and pick smaller candela values? E.g.
```js
const light1 = new THREE.PointLight( 0xff2200, 200 );
light1.position.set( 5, 5, 5 );
scene.add( light1 );

const light2 = new THREE.PointLight( 0x22ff00, 200 );
light2.position.set( - 5, - 5, - 5 );
scene.add( light2 );
```
Just asking since when looking at the PR these high intensity values felt a bit odd.

Reply:
When we enable physically correct lighting, we should probably ensure the scene itself has a realistic scale and lights (specifically point and spot lights) have proper distances to the objects they lit.

Reply:
Values in the 0.5 – 2 range, which we're frequently using in examples now, are not particularly helpful with `useLegacyLights = false`. That's comparable to a candle and (with appropriate exposure) might very dimly illuminate a small room. I might refer to that as a "unitless" workflow, because we're effectively trying to keep light values near 1, disabling attenuation, ignoring exposure, ignoring light units, and trusting that irradiance will generally stay close to the [0,1] range under those conditions.

Hundreds or thousands of candela would be more appropriate for real-world light sources. See [Blender's table of light powers](https://docs.blender.org/manual/en/latest/render/lights/light_object.html#power-of-lights) (approximations), and note that 12.57 lumen = 1 candela, assuming a point light. The conversion is different for other types.

Practically though, this is just a morphing blob, I don't know that we need or want to try to figure out "realistic scale"... we have about 200 examples to update, so I would prefer to keep the changes narrower for now. 😇

Reply:
/cc @WestLangley does this sound correct, and are you OK with starting here?

Reply:
My point is when I would implement the example from scratch with physically correct lighting in mind, I would not position the point lights that far away. I'm aware that candela has a totally different value range than our previous intensity values but even 50000 seems quite high considering the scene and its lighting. 

But yeah, I understand that updating 200 examples is a lot and reconsider distances from lights to their lit objects makes the migration more complex. 

I'm just thinking if we have to touch the examples in any event to make them physically more correct, I would reconsider scale, light distances, too,  and eventually fix #7636.

Reply:
> I'm just thinking if we have to touch the examples in any event to make them physically more correct, I would reconsider scale, light distances, too, and eventually fix https://github.com/mrdoob/three.js/issues/7636.

Although I'm not saying this has to be done all in once. I'm okay with changing the value of `useLegacyLights` and then just update the intensity so the example looks as before. With subsequent PRs, the scene can further be optimized like mentioned above.

Reply:
Remember, three.js is unit-less. Or, to say it another way, the user can assume any units.

If the user wants to use the SI light intensity units defined in the docs, then the user must use _meters_ for distance (1 unit equals 1 meter).

There is a difference between physically-based light _behavior_ (quadratic falloff) and physical light _units_.

Reply:
>const light1 = new THREE.PointLight( 0xff2200, 50000 );

I'd make both point lights have the same intensity... 50000 is unnecessarily bright.

>scene.add( new THREE.AmbientLight( 0x111111 ) );

That is not bright enough to be perceptible.

Reply:
>@WestLangley does this sound correct, and are you OK with starting here?

Merged anyway? Comments ignored.

Reply:
We can always revisit this...

There's a ton of examples to update so we can't afford to have this level of back and forth for each one.

Reply:
Continued in https://github.com/mrdoob/three.js/pull/25479#issuecomment-1482988818.